src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.State.cs

```diff
@@ -63,7 +60,7 @@ private class State
                 /// from.  However, ConditionalWeakTable is the best tool we have, so we simulate a set by just using a
                 /// table and mapping the keys to the <see langword="null"/> value.
                 /// </remarks>
-                public readonly ConditionalWeakTable<ISymbol, object?>? UnrootedSymbolSet;
+                public readonly HashSet<ReferenceHolder<ISymbol>>? UnrootedSymbolSet;
```

Comment:
documentation is now no longer correct.  Also, consider mentioning why we do *not* use a CWT here.

Reply:
Updated